ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/command/toolbar/commands/button/GoalButton.java

```diff
@@ -75,6 +95,34 @@ private void setTooltip(String newTooltipText) {
             tooltip.destroy();
         }
 
-        tooltip = Tooltip.create((Element)getElement(), BOTTOM, MIDDLE, newTooltipText);
+        final DivElement divElement = Elements.createDivElement();
+        divElement.setInnerText(newTooltipText);
+
+        if (keyBinding != null) {
+            final String hotKey = getShortcutText(keyBinding);
+            if (hotKey != null) {
+                SpanElement spanElement = Elements.createSpanElement();
+                spanElement.getStyle().setMarginLeft("5px");
+                spanElement.getStyle().setColor("#aaaaaa");
```

Comment:
White theme is not handled yet.
Will be part of that issue: https://github.com/eclipse/che/issues/4590